man/systemd.unit.xml

```diff
@@ -1023,6 +1023,19 @@
         <xi:include href="version-info.xml" xpointer="v201"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SurviveFinalKillSignal=</varname></term>
+
+        <listitem><para>Takes a boolean argument. Defaults to <option>no</option>. If <option>yes</option>,
+        processes belonging to this unit will not be sent the final <literal>SIGTERM</literal> and
+        <literal>SIGKILL</literal> on state transition (e.g.: shutdown).
+        This is a similar but enhanced mechanism with regards to setting <literal>argv[0][0] = '@'</literal>
+        as described at <ulink url="https://systemd.io/ROOT_STORAGE_DAEMONS">systemd and Storage Daemons for
+        the Root File System</ulink>, that does not require code changes in the executable.</para>
```

Comment:
I'd turn this around, and refer to the root storage thing as legacy. i.e. "This functionality replaces the older bla bla mechanism, which however continues to be supported."